docs/synthetics/bulk-add-monitors-api.md

```diff
@@ -0,0 +1,467 @@
+
+| Monitor Name | URL | Locations | Interval
+| --- | --- | --- | --- |
+| bulk add 1 | https://example_1.com | 1, 2 | 5 |
+| bulk add 3 | https://example_3.com | 3, 4 | 10 |
+
+When iterating through each monitor, extract the location and interval settings for that specific monitor and add them as parameters in the payload.
+
+```javascript
+for (var i = 2; i <= sheet.getLastRow(); i++) {
+    var name = String(sheet.getRange(i, 1).getValue()).trim();
+    var url = String(sheet.getRange(i, 2).getValue()).trim();  
+    var locations = String(sheet.getRange(i, 3).getValue()).trim();
+    var intervalValue = String(sheet.getRange(i, 4).getValue()).trim();
+    var interval = intervalValue + "m"; // Add "m" to interval
+
+    // Convert locations from "1, 2, 3" to [1, 2, 3]
+    var locationArray = locations.split(",").map(function(loc) {
+      return parseInt(loc.trim(), 10);
+    }).filter(function(num) {
+      return !isNaN(num);
+    });
+
+    if (!name || !url) {
+      Logger.log("Skipping row " + i + " due to missing data.");
+      continue;
+    }
+    if (!url.startsWith("http://") && !url.startsWith("https://")) {
+      Logger.log("ERROR at row " + i + ": Invalid URL format.");
+      continue;
+    }
+    // Define the configuration parameters for creating an HTTP monitor.
+    var payload = {
+      "name": name,
+      "interval": interval,
+      "enabled": true,
+      "locations": locationArray,
+      "url": url,
+      "method": "GET",
+...
+```
+
+### Run Apps Script
+
+After creating your Google Sheet and writing the Apps Script based on your requirements, click **Run** to create the monitors. You will see the execution log below. 
+
+![Synthetics Apps Script](../images/synthetics/synthetics-api-apps-script.png)
+
+Then, go to your **Sematext → Synthetics App**, where you should see the HTTP monitors created in your Sematext account.
+
+![Synthetics API Bulk Add Monitors](../images/synthetics/synthetics-api-bulk-monitors.png)
+
+## Bulk Edit Monitors
+
+After bulk adding monitors, if you want to change the configuration for each one, you can use the bulk edit API. Note that you need to pass each parameter again because the edit API overrides the existing configuration. To bulk edit all the monitors, you can clone the spreadsheet and make the necessary changes.
+                                                                                                                  
+> If you want to bulk edit Browser monitors, follow these steps first, but refer to the [Bulk Edit Browser Monitors](#bulk-edit-browser-monitors) section below for the minor changes needed to edit Browser monitors.
+
+For example, to add one more location to all the monitors created by [this script](#setting-up-the-google-sheet-to-bulk-add-monitors).  Clone the sheet and change the App Script and the Google Sheet according to the instructions below:
```

Comment:
I think that ".   Clone" is a mistake. Probably didn't want to end the sentence there?